yak_ros/include/yak_ros/utils.h

```diff
@@ -69,8 +70,11 @@ bool updateParams(kfusion::KinFuParams& params, yak_ros_msgs::UpdateKinFuParamsR
     }
     else if (param == request.VOLUME_POSE)
     {
-      // I think it is a bad idea to let people change this.
-      return false;
+      Eigen::Isometry3d mid;
+      tf::poseMsgToEigen(request.kinfu_params.volume_pose, mid);
+      params.volume_pose= cv::Affine3f::Identity().translate(cv::Vec3f(mid.translation().x(), mid.translation().y(), mid.translation().z()));
```

Comment:
Is there a more direct way to perform this conversion?

Reply:
May be able to use `#include <opencv2/core/eigen.hpp>.`